scripts/0-generate-viai-application-assets.sh

```diff
@@ -87,7 +87,7 @@ CONTAINER_REPO_TYPE=""
 GOOGLE_CLOUD_PROJECT=
 K8S_RUNTIME=
 VIAI_CLIENT_INTEGRATION_SERVICE_ACCOUNT_KEY_PATH=
-VIAI_CAMERA_INTEGRATION_SOURCE_REPO_URL=https://github.com/GoogleCloudPlatform/solutions-viai-edge-camera-integration
+VIAI_CAMERA_INTEGRATION_SOURCE_REPO_URL=https://source.developers.google.com/p/cloud-ce-shared-csr/r/MARKKU-viai-edge-camera-integration
```

Comment:
Is this the correct URL? I'd not hard code this, but let the user choose.

Reply:
yes. we decided to use this CE source repositories as the source repository for partners / customers.

Reply:
revert back to public repo url per discussion